templates/index.mustache

```diff
@@ -45,7 +45,7 @@
                     <p data-i18n-text="Interested in automating your preferred strategies for trading binary options?"></p>
     				<p data-i18n-text="Binary Bot is our leading-edge programming tool which allows you to build trading apps with a simple “jigsaw puzzle-like” drag-and-drop function."></p>
                     <p data-i18n-text="Dream up any number of binary options trading bots, from incredibly simple formulas to vastly complex algorithms. Then let them trade for you, even while you sleep."></p>
-    				<p><span data-i18n-text="Best of all, once you’ve built a bot, we can help you sell it to fellow traders in the"></span> <a href="https://shop.binary.com" target="_blank" data-i18n-text="Binary.com Shop"></a>. <span data-i18n-text="Receive 80% of the sales revenue from every purchase of your bot in our app store."></span></p>
+    				<p><span data-i18n-text="Best of all, once you’ve built a bot, we can help you sell it to fellow traders in the"></span> <a id="shop-url" href="https://shop.binary.com/collections/strategies" target="_blank" data-i18n-text="Binary.com Shop"></a>. <span data-i18n-text="Receive 80% of the sales revenue from every purchase of your bot in our app store."></span></p>
```

Comment:
We can have this as a temporary solution, in the future it is better to make this as a generic component. Something similar to 
<ExtensionAnchor id="..." href="..." data-i18n-text="..." />
And replace the 'href' in componendDidMount() event using RegExp. So we do not have to update URL for each a tag.